js/src/views/Status/components/Debug/Debug.css

```diff
@@ -37,7 +37,9 @@
 }
 
 .logs {
```

Comment:
I guess the `.log` class isn't used anymore ; get rid of it?
And also now the logs are white, they used to be gray (`#aaa`)